javalib/src/main/scala/java/io/File.scala

```diff
@@ -298,7 +298,7 @@ class File(_path: String) extends Serializable with Comparable[File] {
       )
       fromCWideString(resolvedName, StandardCharsets.UTF_16LE)
     } else {
-      val resolvedName = alloc[Byte](limits.PATH_MAX)
+      val resolvedName: Ptr[Byte] = alloc[Byte](limits.PATH_MAX)
```

Comment:
@WojciechMazur Why is the type required here?

Reply:
When we would remove `alloc[T]` method this workaround would no longer be needed. Alternatively we can not include this method only in Scala3 unsafe package, but then breaking both binary and source compatibility does not sound very nice, especially when someone is using a lot of `stackalloc[T]` or `alloc[T]` calls

Reply:
Thank-you so much for explaining - sorry I didn't review prior. Hard to keep track given your velocity.